docs/unified/novacustom/fan-profiles.md

```diff
@@ -0,0 +1,23 @@
+# Fan profiles settings
+
+There are two fan profiles implemented. The profiles can be selected via the
+[Power Management Options](/dasharo-menu-docs/dasharo-system-features/#power-management-options)
+menu in Setup Menu.
+
+## Silent fan profile
+
+| Temperature [°C] | Fan speed [%] |
+|------------------|---------------|
+| 0                | 25            |
+| 65               | 30            |
+| 75               | 35            |
+| 100              | 100           |
+
+## Performance fan profile
+
+| Temperature [°C] | Fan speed [%] |
+|------------------|---------------|
+| 0                | 25            |
+| 55               | 35            |
+| 75               | 60            |
+| 100              | 100           |
```

Comment:
@macpijan What is the characteristic between the indicated points? Linear? I think such information would be helpful for users.

Reply:
Values inbetween curve points are linearly interpolated, yes.

For temperatures below the first point the fans will be stoppend and above the last point the fans will be set to 100%.

Reply:
So we should add such an information

Reply:
Added a note about this.